integration/kubernetes/k8s-health-checks.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	get_pod_config_dir
+}
+
+@test "Health checks" {
+	pod_name="hctest"
+	bad_pod_name="badhctest"
+	wait_time=20
+	sleep_time=2
+
+	# Create pod
+	kubectl create -f "${pod_config_dir}/pod-health.yaml"
+
+	# Check pod creation
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	# Create a bad pod
+	kubectl create -f "${pod_config_dir}/pod-bad-health.yaml"
+
+	# Check pod creation
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	# Check bad pod status
+	cmd="kubectl describe pod $bad_pod_name | grep Unhealthy"
```

Comment:
I'm slightly confused, as it looks like we create a healthy pod, and then an unhealthy pod, but only check the status for `Unhealthy`. Should we also be checking the healthy one is actually healthy as well?

Reply:
@GabyCT Maybe I am missing something, but I still dont see how these environment variables are used?
Are they used by the workload itself?

Reply:
Agree... maybe a little explanation of how the service works to check for the health of the pod?